src/component/editor/config/data.js

```diff
@@ -0,0 +1,405 @@
+export const CHARACTERS = [
```

Comment:
This is a `.js` file and I thought we prohibit that.

Reply:
resolved